sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -2136,98 +2138,128 @@ case class GpuStringInstr(str: Expression, substr: Expression)
   }
 }
 
-class GpuConvMeta(
-  expr: Conv,
-  conf: RapidsConf,
-  parent: Option[RapidsMeta[_,_,_]],
-  rule: DataFromReplacementRule) extends TernaryExprMeta(expr, conf, parent, rule) {
+case class GpuConv(num: Expression, fromBase: Expression, toBase: Expression, ansiEnabled: Boolean)
+  extends GpuTernaryExpression {
 
-  override def tagExprForGpu(): Unit = {
-    val fromBaseLit = GpuOverrides.extractLit(expr.fromBaseExpr)
-    val toBaseLit = GpuOverrides.extractLit(expr.toBaseExpr)
-    val errorPostfix = "only literal 10 or 16 are supported for source and target radixes"
-    (fromBaseLit, toBaseLit) match {
-      case (Some(Literal(fromBaseVal, IntegerType)), Some(Literal(toBaseVal, IntegerType))) =>
-        def isBaseSupported(base: Any): Boolean = base == 10 || base == 16
-        if (!isBaseSupported(fromBaseVal) && !isBaseSupported(toBaseVal)) {
-          willNotWorkOnGpu(because = s"both ${fromBaseVal} and ${toBaseVal} are not " +
-            s"a supported radix, ${errorPostfix}")
-        } else if (!isBaseSupported(fromBaseVal)) {
-          willNotWorkOnGpu(because = s"${fromBaseVal} is not a supported source radix, " +
-            s"${errorPostfix}")
-        } else if (!isBaseSupported(toBaseVal)) {
-          willNotWorkOnGpu(because = s"${toBaseVal} is not a supported target radix, " +
-            s"${errorPostfix}")
-        }
-      case _ =>
-        // This will never happen in production as the function signature enforces
-        // integer types for the bases, but nice to have an edge case handling.
-        willNotWorkOnGpu(because = "either source radix or target radix is not an integer " +
-          "literal, " + errorPostfix)
+  override def doColumnar(
+    strCv: GpuColumnVector,
+    fromCv: GpuColumnVector,
+    toCv: GpuColumnVector): ColumnVector = {
+    if (ansiEnabled &&
+      NumberConverter.isConvertOverflowCvCvCv(strCv.getBase, fromCv.getBase, toCv.getBase)) {
+      ConvUtils.overflowInConvError()
     }
+    NumberConverter.convertCvCvCv(strCv.getBase, fromCv.getBase, toCv.getBase)
```

Comment:
Do we have tests to cover all nulls case ? e.g. at least one of strCv, fromCv and toCv has all null rows.